include/deal.II/base/parameter_handler.h

```diff
@@ -2343,8 +2388,17 @@ class MultipleParameterLoop : public ParameterHandler
    * ParameterHandler, this is the only one overridden with new behavior by
    * this class. This is because the other two <tt>read_input</tt> functions
    * just reformat their inputs and then call this version.
-   */
-  virtual bool read_input (std::istream &input,
+   *
+   * @deprecated This function used to report success or failure by printing
+   * messages to <code>std::cerr</code> and returning a boolean
```

Comment:
This comment is now no longer accurate. I think you probably just forgot to remove it again. (If you intended to keep it: there is generally little value in function-level documentation to talk about the *history* of the function's implementation because users don't usually care about that. They care about what the function *now* does.)